src/PortingAssistantExtensionServer/Program.cs

```diff
@@ -34,11 +33,15 @@ static async Task Main(string[] args)
                 var stdInPipeName = args.Length == 1 ? Common.Constants.stdDebugInPipeName : args[1];
                 var stdOutPipeName = args.Length == 1 ? Common.Constants.stdDebugOutPipeName : args[2];
                 Common.PALanguageServerConfiguration.ExtensionVersion = args.Length == 1 ? "0.0.0" : args[3];
+                var vsClientVersion = args.Length == 1 ? "unkwown" : args[4];
```

Comment:
should you use your constant for vs_unknown here? also there's a typo haha

Reply:
thanks, will fix that